src/ape/api/providers.py

```diff
@@ -136,10 +136,24 @@ def decode(cls, data: dict) -> "ReceiptAPI":
         ...
 
 
+@abstractdataclass
+class BlockGasAPI:
+    raw_data: Dict
```

Comment:
still really love Pydantic for here, because you can do `__root__: dict` to have it work like a dict with extra methods and validatorss